cmd/key_describe.go

```diff
@@ -0,0 +1,139 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"code.vegaprotocol.io/vegawallet/cmd/cli"
+	"code.vegaprotocol.io/vegawallet/cmd/flags"
+	"code.vegaprotocol.io/vegawallet/cmd/printer"
+	"code.vegaprotocol.io/vegawallet/wallet"
+	"code.vegaprotocol.io/vegawallet/wallets"
+	"github.com/spf13/cobra"
+)
+
+var (
+	describeKeyLong = cli.LongDesc(`
+		Describe the all known information about the specified key pair
```

Comment:
English is not my first language, but I'm not sure `Describe the all know information` is what you wanted to write?

Reply:
It certainly was not!